docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -220,53 +244,90 @@ pip install -v -e .
-    make -j$(nproc) && make install
-    ```
+```shell
+mkdir -p build && cd build
+cmake .. \
+    -DMMDEPLOY_BUILD_SDK=ON \
+    -DMMDEPLOY_BUILD_SDK_PYTHON_API=ON \
+    -DMMDEPLOY_TARGET_DEVICES="cuda;cpu" \
+    -DMMDEPLOY_TARGET_BACKENDS="trt" \
+    -DMMDEPLOY_CODEBASES=all \
+    -Dpplcv_DIR=${PPLCV_DIR}/cuda-build/install/lib/cmake/ppl
+make -j$(nproc) && make install
+```
+
+```{note}
+It takes about 9 minutes to build SDK libraries on a Jetson Nano. So, please be patient until the installation is complete.
+```
 
 2. Build SDK demos
 
-    ```shell
-    cd ${MMDEPLOY_DIR}/build/install/example
-    mkdir -p build && cd build
-    cmake .. -DMMDeploy_DIR=${MMDEPLOY_DIR}/build/install/lib/cmake/MMDeploy
-    make -j$(nproc)
-    ```
+```shell
+cd ${MMDEPLOY_DIR}/build/install/example
+mkdir -p build && cd build
+cmake .. -DMMDeploy_DIR=${MMDEPLOY_DIR}/buildinstall/lib/cmake/MMDeploy
+make -j$(nproc)
+```
+
+### Run a Demo
 
-3. Run a demo
+#### Object Detection demo
 
-    Take the object detection for example:
-    ```shell
-    ./object_detection cuda ${directory/to/the/converted/models} ${path/to/an/image}
-    ```
+Before running this demo, you need to convert model files to be able to use with this SDK.
+
+1. Install [MMDetection](https://github.com/open-mmlab/mmdetection) which is needed for model conversion
+
+MMDetection is an open source object detection toolbox based on PyTorch
+
+```shell
+git clone https://github.com/open-mmlab/mmdetection.git
+cd mmdetection
+pip install -r requirements/build.txt
+pip install -v -e .  # or "python setup.py develop"
+```
+
+2. Follow [this document](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/tutorials/how_to_convert_model.md) on how to convert model files.
+
+For this example, we have used [retinanet_r18_fpn_1x_coco.py](https://github.com/open-mmlab/mmdetection/blob/master/configs/retinanet/retinanet_r18_fpn_1x_coco.py) as the model config, and [this file](https://download.openmmlab.com/mmdetection/v2.0/retinanet/retinanet_r18_fpn_1x_coco/retinanet_r18_fpn_1x_coco_20220407_171055-614fd399.pth) as the corresponding checkpoint file. Also for deploy config, we have used [detection_tensorrt_dynamic-320x320-1344x1344.py](https://github.com/open-mmlab/mmdeploy/blob/master/configs/mmdet/detection/detection_tensorrt_dynamic-320x320-1344x1344.py)
+
+```shell
+python ./tools/deploy.py \
+    configs/mmdet/detection/detection_tensorrt_dynamic-320x320-1344x1344.py \
+    $PATH_TO_MMDET/configs/retinanet/retinanet_r18_fpn_1x_coco.py \
+    retinanet_r18_fpn_1x_coco_20220407_171055-614fd399.pth \
+    $PATH_TO_MMDET/demo/demo.jpg \
+    --work-dir work_dir \
+    --show \
+    --device cuda:0 \
+    --dump-info
+```
+
+3. Finally run inference on an image
+
+```shell
+./object_detection cuda ${directory/to/the/converted/models} ${path/to/an/image}
+```
+
+<div align=center><img width=1000 src="https://files.seeedstudio.com/wiki/open-mmlab/output_detection.png"/></div>
```

Comment:
May consider putting a source image before the detection result.

Reply:
What do you mean by source image?

Reply:
The image with no detection results drawn.

Reply:
oh okay. let me update now

Reply:
It is done